integration/tests/navigation.test.ts

```diff
@@ -91,129 +91,130 @@ export default function Page() {
-  //
-  //   await u.page.getByText(/Manage account/).click();
-  //
-  //   await u.page.waitForSelector('.cl-modalContent > .cl-userProfile-root', { state: 'attached' });
-  //
-  //   await u.page.getByText(/Set username/i).click();
-  //   await u.page.getByText(/Cancel/i).click();
-  //
-  //   await u.page.getByText(/Add email address/i).click();
-  //   await u.page.getByText(/Cancel/i).click();
-  // });
-  //
-  // test('sign in with path routing navigates to previous page', async ({ page, context }) => {
-  //   const u = createTestUtils({ app, page, context });
-  //   await u.po.signIn.goTo();
-  //   await u.po.signIn.waitForMounted();
-  //
-  //   await u.po.signIn.getGoToSignUp().click();
-  //   await u.po.signUp.waitForMounted();
-  //   await u.page.waitForURL(`${app.serverUrl}/sign-up?redirect_url=${encodeURIComponent(app.serverUrl + '/')}`);
-  //
-  //   await page.goBack();
-  //   await u.po.signIn.waitForMounted();
-  //   await u.page.waitForURL(`${app.serverUrl}/sign-in`);
-  // });
+  test('user profile with path routing', async ({ page, context }) => {
+    const u = createTestUtils({ app, page, context });
+    await u.po.signIn.goTo();
+    await u.po.signIn.waitForMounted();
+    await u.po.signIn.signInWithEmailAndInstantPassword({ email: fakeUser.email, password: fakeUser.password });
+    await u.po.expect.toBeSignedIn();
+
+    await u.po.userProfile.goTo();
+    await u.po.userProfile.waitForMounted();
+
+    await u.po.userProfile.clickSetUsername();
+
+    u.page.getByText(/Update username/i);
+
+    await u.po.userProfile.typeUsername('some_username');
+
+    await u.page.getByText(/Cancel/i).click();
+
+    await u.page.waitForSelector('.cl-profileSectionContent__username .cl-headerTitle', { state: 'detached' });
+
+    await u.po.userProfile.clickAddEmailAddress();
+
+    u.page.getByText(/an email containing/i);
+
+    await u.po.userProfile.typeEmailAddress('some@email.com');
+
+    await u.page.getByText(/Cancel/i).click();
+  });
+
+  test('user profile with hash routing', async ({ page, context }) => {
+    const u = createTestUtils({ app, page, context });
+    await u.po.signIn.goTo();
+    await u.po.signIn.waitForMounted();
+    await u.po.signIn.signInWithEmailAndInstantPassword({ email: fakeUser.email, password: fakeUser.password });
+    await u.po.expect.toBeSignedIn();
+
+    await u.page.goToRelative('/hash/user');
+    await u.po.userProfile.waitForMounted();
+
+    await u.po.userProfile.clickSetUsername();
+
+    u.page.getByText(/Update username/i);
+
+    await u.po.userProfile.typeUsername('some_username');
+
+    await u.page.getByText(/Cancel/i).click();
+
+    await u.page.waitForSelector('.cl-profileSectionContent__username .cl-headerTitle', { state: 'detached' });
+
+    await u.po.userProfile.clickAddEmailAddress();
+
+    u.page.getByText(/an email containing/i);
+
+    await u.po.userProfile.typeEmailAddress('some@email.com');
+
+    await u.page.getByText(/Cancel/i).click();
```

Comment:
This test no longer tests navigation but interaction within the component. Should this be moved ?